<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VErrorStatistics</class>
 <widget class="QWidget" name="VErrorStatistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1750</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblClearErrorCount">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Count of Errors:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bntClear">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bntExport">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>10</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>1731</width>
     <height>601</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ErrorCode</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Count</string>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </column>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
